docs/built-in-models/nearest-neighbors.md

```diff
@@ -0,0 +1,21 @@
+# Nearest-neighbors
+
+Concrete ML offers nearest neighbors non-parametric classification models with a scikit-learn interface through the `KNeighborsClassifier` class.
+
+|        Concrete ML         | scikit-learn                                                                                                          |
+| :------------------------: | --------------------------------------------------------------------------------------------------------------------- |
+| [KNeighborsClassifier](<>) | [KNeighborsClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html) |
+
+## Example usage
+
+```python
+from concrete.ml.sklearn import KNeighborsClassifier
+
+concrete_classifier = KNeighborsClassifier(n_bits=2,     n_neighbors=3)
+```
+
```

Comment:
I suggest:

The `KNeighborsClassifier` class quantizes quantization on the training data-set provided to the .fit method, with the specified number of bits, `n_bits`. It is crucial to maintain a low value for n_bits to comply to the accumulator size constraints and the algorithm's time complexity, which is closely tied to three key factors: n_bits, data dimensionality, and the size of the training dataset.

The Concrete K-nearest neighbors computation operates in O(Nlog⁡2k) steps, where $$N$$ is the training dataset size and $$k$$ is `n_neighbors`. Furthermore, the size of the dataset has a linear impact on the complexity of the data and the number of nearest neighbors, `n_neighbors`, also plays a role.